Client/DataAccess/SQL/VisitsTable.swift

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+let TableNameVisits = "visits"
+
+class VisitsTable<T>: GenericTable<Visit> {
+    override var name: String { return TableNameVisits }
+    override var rows: String { return "guid TEXT NOT NULL UNIQUE, " +
+                      "siteGuid TEXT NOT NULL, " +
+                      "date REAL NOT NULL" }
```

Comment:
I think there are a couple of changes here:

* Visits probably don't need a GUID.
* The place reference ("site" might be misleading?) should probably be a foreign key reference to the numeric ID of the place.
* Visits should have a type (see Places docs).